src/openrct2/scripting/ScriptEngine.cpp

```diff
@@ -771,6 +777,44 @@ DukValue ScriptEngine::PositionToDuk(const CoordsXYZ& position)
     return DukValue::take_from_stack(ctx);
 }
 
+constexpr static const char* ExpenditureTypes[] = { "ride_construction",
+                                                    "ride_runningcosts",
+                                                    "land_purchase",
+                                                    "landscaping",
+                                                    "park_entrance_tickets",
+                                                    "park_ride_tickets",
+                                                    "shop_sales",
+                                                    "shop_stock",
+                                                    "food_drink_sales",
+                                                    "food_drink_stock",
+                                                    "wages",
+                                                    "marketing",
+                                                    "research",
+                                                    "interest" };
```

Comment:
If you put an trailing comma and put "ride_construction" on the next line, will it stay in shape after clang-format? I find this looking quite ugly and hard to read.

Reply:
I can't get it to do what you are suggesting. Are you able to make change?